Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -737,6 +691,17 @@ void EmbeddedFrag::addAtomToAtomWithAng(unsigned int aid, unsigned int toAid) {
   rtrans.SetTransform(refLoc, currAngle);
   RDGeom::Point2D currLoc = nb2;
   rtrans.TransformPoint(currLoc);
+  if (fabs(remAngle) - M_PI < 1e-3) {
```

Comment:
Is this clash detection?

Reply:
This detects the condition which could lead to the clashes.
The clashes themselves are caught in the calls to `findNumNeigh()` a couple lines down